hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/handshakeserver/HandshakeServer.java

```diff
@@ -82,13 +87,15 @@ public void run() {
 							if (socketInput.available() > 0) {
 								byte[] message = new byte[44];
 								socketInput.read(message);
-								extractData(message);
-								logger.info("Completed reading SI Segment");
+								String siSegment = extractData(message);
+								logger.info(String.format("Received from originator %s byte SI Data Block:: %s",message.length,ret_code));
+								logger.info(String.format("Received SI segment: %s", siSegment));
```

Comment:
More like 'debug' kind of log

Reply:
Anu mentioned, this is done to keep consistent with legacy logs. So resolving this one